### ISSUE TITLE
Watermark shows solid white rectangular when opacity != 100%

### ISSUE THREAD

**Issue by dsync89** [ErsatzTV/ErsatzTV#1856]
When setting the watermark opacity to any value lower than 100%, it will only show a white solid rectangle.

![image](https://github.com/user-attachments/assets/51cdd110-7758-405c-aa5a-fb1e868143b7)

![image](https://github.com/user-attachments/assets/d89ad78d-4e75-4296-a6b9-02ea8e45652e)

Image source:

![discovery-channel-icon-hd-us](https://github.com/user-attachments/assets/d400981b-fc1c-4066-a44e-804f643db1bc)

**Comment by jasongdove** [comment]
Please paste your ETV version found below the left menu.

Will also need to know what your ffmpeg profile looks like. You can either screenshot or paste from the support > troubleshooting page, general section.

**Comment by dsync89** [comment]
Sure @jasongdove I'm using version `0.8.7-55b7a356-docker-vaapi`.

This is my ffmpeg profile:
```
  "FFmpegSettings": {
    "FFmpegPath": "/usr/local/bin/ffmpeg",
    "FFprobePath": "/usr/local/bin/ffprobe",
    "DefaultFFmpegProfileId": 1,
    "PreferredAudioLanguageCode": "eng",
    "UseEmbeddedSubtitles": true,
    "ExtractEmbeddedSubtitles": false,
    "SaveReports": false,
    "HlsSegmenterIdleTimeout": 30,
    "WorkAheadSegmenterLimit": 1,
    "InitialSegmentCount": 1,
    "HlsDirectOutputFormat": "Mp4"
  },
```

![image](https://github.com/user-attachments/assets/253e77ac-37ad-45db-853d-19545d7316b6)

**Comment by jasongdove** [comment]
The FFmpeg Profile is actually on the page just below `Channels` called `FFmpeg Profiles`. Click the edit button on whichever one you are using; that contains the information that I need.

**Comment by dsync89** [comment]
@jasongdove Okay here's the screenshot as requested.

![image](https://github.com/user-attachments/assets/4c7aaccf-0071-41ad-b13a-8a7025e18ac2)

**Comment by jasongdove** [comment]
@dsync89 one thing that stands out to me is that the watermark editor screenshot does not show the watermark. Do you have the original file to try re-uploading as a new watermark? There was a bug in old ETV versions that mangled certain image uploads, fixed by https://github.com/ErsatzTV/ErsatzTV/pull/1569 and released in v0.8.5-beta.

**Comment by jasongdove** [comment]
Still curious about why the image doesn't display in that editor, but I can confirm the bug when modifying watermark opacity with an image that already has an alpha channel.
